wifiphisher/pywifiphisher.py

```diff
@@ -158,7 +170,21 @@ def parse_args():
         help="Force the usage of hostapd installed in the system",
         action='store_true')
 
-    return parser.parse_args()
+    parser.add_argument("-pPD",
+                        "--phishing-pages-directory",
+                        help="Search for phishing pages in this location")
+    parser.add_argument(
+        "-dC",
+        "--dnsmasq-conf",
+        help="Determine the full path of a custom dnmasq.conf file",
+        default='/tmp/dnsmasq.conf')
+    parser.add_argument(
+        "-cP",
+        "--credential-log-path",
+        help="Determine the full path of the file that will store any captured credentials",
+        default=None)
+
```

Comment:
Question: The `-cP` argument should be used in conjunction with the `--logging`, right? I think you should add this check under the check_args method: https://github.com/wifiphisher/wifiphisher/blob/master/wifiphisher/common/opmode.py#L56

Also, the `-pPD` option does not make sense to be used with `-iI` (internet mode). Maybe you should add a check about it too.

Reply:
> Also, the -pPD option does not make sense to be used with -iI (internet mode). Maybe you should add a check about it too.

I've added the checks for the logging and the save position.
One thing is it not clear, why i would not want to have a connected interface and custom phishing scenarios?

Reply:
Oups, sorry I meant the `-lP` should be used in conjunction with the `--logging` only. The `-cP` can be used on its own.

If you use the `-iI` option, then you typically allow the victim to browse the Internet in order to sniff for credentials. There's no phishing included there (yet).